common/logger/src/lib.rs

```diff
@@ -72,7 +72,8 @@
 
 pub mod prelude {
     pub use crate::{
-        debug, error, event, info, security::SecurityEvent, trace, warn, StructuredLogEntry,
+        debug, error, event, info, libra_logger::FileWriter, security::SecurityEvent, trace, warn,
```

Comment:
Nit: I don't think we need to provide the FileWriter to all uses of the prelude.  But, nbd

Reply:
I'd have to do some wonky stuff to export it. I think the right approach is to probably move the writers to their own file and allow that to be exported.